src/lib/tabs/tabs.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {Component} from '@angular/core';
```

Comment:
Not sure why the extra test is needed, isn't this covered by `tab-group.spec.ts`?

Reply:
It was not caught by the current tests in `tab-group.spec.ts`, would you prefer I move it into `tab-group.spec.ts`?